doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -347,7 +347,13 @@ Bluetooth libraries
   * Added support for the GATT API serialization.
   * Changed the configuration option that enables the library from the :kconfig:`CONFIG_BT_RPC` to the :kconfig:`CONFIG_BT_RPC_STACK`.
 
-Added:
+* :ref:`bt_mesh` library:
```

Comment:
I moved those three bullets to mesh library.
To me it sounded like it's where they belong.
Open for discussion.

Reply:
I kind of agree, but this appears to fit with the rest of the structure in the document, at least

Reply:
I agree as well, and I would prefer to have one bit section for Protocols. This said, we have to follow the modular structure principle for the NCS, and if we merge Protocols with Samples, there will be sections for Apps, Libs, Drivers, Scripts, but no section for Samples, which will be weird. Also, some samples are not bound to one specific protocol, but to libraries, and so there must be a compromise we must settle on for this. :/